junit-platform-engine/README.md

```diff
@@ -246,7 +258,7 @@ Supported `DiscoverySelector`s are:
 The only supported `DiscoveryFilter` is the `PackageNameFilter` and only when
```

Comment:
What is a `DiscoverySelector`? It might be nice to explain first before listing the supported ones.

Reply:
Do you linking to this is sufficient:

https://junit.org/junit5/docs/current/user-guide/#advanced-topics

Reply:
Perhaps. I was thinking more along the lines of a short intro to this section (why would I care about this?), as it seemed a bit out of left field to me. I've added a short intro; feel free to improve.

Reply:
Ah I see. The selectors let you select the scenario/feature you want to run. This is useful for people who run their test through JUnits API or people who develop IDE plugins that integrate Cucumber.